### PR TITLE
Repackage Recorded Future

### PR DESCRIPTION
**Pull request by v-atulyadav**
Required items, please complete
   
   Change(s):
   - See guidance below

   Reason for Change(s):
   - See guidance below

   Version Updated:
   - Required only for Detections/Analytic Rule templates
   - See guidance below

   Testing Completed:
   - [Test Doc](https://microsoftapc.sharepoint.com/:w:/r/teams/EcosystemCATTeam/_layouts/15/Doc.aspx?sourcedoc=%7B99CB90EF-F09E-4C20-A5C8-56AA0EBAE298%7D&file=Sentinel%20Release%20Checklist%20-V2%20solutions.docx&action=default&mobileredirect=true&share=IQHvkMuZnvAgTKXIVqoOuuKYAT33ZZ_Yl28mtNDRn-K_kl0)

   Checked that the validations are passing and have addressed any issues that are present:
   - See guidance below

# Guidance <- remove section before submitting
-----------------------------------------------------------------------------------------------------------
## Before submitting this PR please ensure that you have read the following sections and filled out the changes, reason for change and testing complete sections:

Thank you for your contribution to the Microsoft Sentinel Github repo.

> Details of the code changes in your submitted PR.  Providing descriptions for pull requests ensures there is context to changes being made and greatly enhances the code review process.  Providing associated Issues that this resolves also easily connects the reason.
   
   Change(s):
   - Updated syntax for XYZ.yaml

   Reason for Change(s):
   - New schema used for XYZ.yaml
   - Resolves ISSUE #1234

   Version updated:
   - Yes
   - Detections/Analytic Rule templates are required to have the version updated

> The code should have been tested in a Microsoft Sentinel environment that does not have any custom parsers, functions or tables, so that you validate no incorrect syntax and execution functions properly.  If your submission requires a custom parser or function, it must be submitted with the PR.

   Testing Completed:
   - Yes/No/Need Help

_Note: If updating a detection, you must update the version field._

> Before the submission has been made, please look at running the KQL and Yaml Validation Checks locally.
> https://github.com/Azure/Azure-Sentinel#run-kql-validation-locally

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes/No/Need Help
   
   _Note: Let us know if you have tried fixing the validation error and need help._

> References: 
> - [Guidance for Detection checks](https://github.com/Azure/Azure-Sentinel#pull-request-detection-template-structure-validation-check)
> - [General contribution guidance](https://github.com/Azure/Azure-Sentinel/wiki#what-can-you-contribute-and-how-can-you-create-contributions)
> - [PR validation troubleshooting](https://github.com/Azure/Azure-Sentinel#pull-request)

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

**Comment by v-dvedak** [comment]
Working on it

File: Solutions/Recorded Future/Package/createUiDefinition.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
...
+        "description": "<img src=\"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Logos/RecordedFuture.svg\" width=\"75px\" height=\"75px\">\n\n**Note:** _There may be [known issues](https://aka.ms/sentinelsolutionsknownissues) pertaining to this Solution, please refer to them before installing._\n\n[Recorded Future](https://www.recordedfuture.com/) is the worlds largest provider of intelligence for enterprise security. By combining persistent and pervasive automated data collection and analytics with human analysis, Recorded Future delivers intelligence that is timely, accurate, and actionable.\n\n**Workbooks:** 2, **Analytic Rules:** 6, **Playbooks:** 6\n\n[Learn more about Microsoft Sentinel](https://aka.ms/azuresentinel) | [Learn more about Solutions](https://aka.ms/azuresentinelsolutionsdoc)",
```
Comment (v-sabiraj) [inline Solutions/Recorded Future/Package/createUiDefinition.json:9]: @v-atulyadav, please check on the description and update, thanks.

**Comment by RecordedFutureOskbo** [comment]
This will conflict with updates in PR https://github.com/Azure/Azure-Sentinel/pull/6307#issuecomment-1327031038. We plan to look over the texts and update instructions and include pictures in the future. I'm just waiting for the 6307 PR to merge.

**Comment by v-sabiraj** [comment]
Hey @RecordedFutureOskbo, please let us know once that PR is merged, so that we can resolve conflicts and merge this as well, thanks.

**Comment by v-sabiraj** [comment]
Hey @v-mchatla, can you please merge https://github.com/Azure/Azure-Sentinel/pull/6307#issuecomment-1327031038 , this Pr so that we can proceed with this, thanks.

**Comment by v-sabiraj** [comment]
@v-atulyadav, as the changes from other PR are merged can you check and resolve the conflicts, thanks.

**Comment by v-sabiraj** [comment]
@v-atulyadav, as the changes from other PR are merged can you check and resolve the conflicts, thanks.

**Pull request closed without merging**